README.md

```diff
@@ -7,23 +7,37 @@ as part of [Desert Bus For Hope](https://desertbus.org).
 A full design doc can be read at [initial-design-doc.pdf](./initial-design-doc.pdf),
 but a brief overview of the components:
 
+#### Shared Components
+
+* `common` provides code shared between the other components.
+
+#### Ingest
 * `downloader` grabs segments from twitch and saves them to disk
 * `restreamer` serves segments from disk as well as playlist files allowing them to be streamed
 * `backfiller` queries restreamers of other servers in order to pick up segments this server doesn't have already,
   ie. it replicates missing segments.
-* `cutter` interacts with a database to perform cutting jobs
+* `chat_archiver` records twitch chat messages and merges them with records from other nodes.
+
+#### Processing
 * `sheetsync` syncs specifc database columns to a google doc which is the primary operator interface.
+* `cutter` interacts with a database to perform cutting jobs
 * `thrimshim` acts as an interface between the `thrimbletrimmer` editor and the database.
 * `thrimbletrimmer` is a browser based video editor.
 * `segment_coverage` regularly checks whether there is complete segment coverage for each hour. 
```

Comment:
I would include segment_coverage under Ingest since it is something run on each of the replication nodes

Reply:
segment_coverage probably belongs under analysis, as it doesn't involve actually grabbing data from somewhere.

Or the other option is to completely restructure this section into "replication nodes", "edit nodes" etc. Which may be more useful.